xml/cap_admin_minibroker.xml

```diff
@@ -22,8 +22,8 @@
   <link xlink:href="https://github.com/osbkit">Microsoft Azure team</link>. It
   provides a simple method to provision service brokers on Kubernetes clusters.
  </para>
- <important>
-  <title>Minibroker Upstream Services</title>
+ <warning>
+  <title>Do Not Use Minibroker on Production Systems</title>
```

Comment:
See above comment. These 2 lines should be removed. (They're also the cause of the Travis builds failing)

Reply:
I do not understand what should I change exactly. We can talk 1-1 to understand it better.
What I get so far is that I should remove the lines marked with red, and keep the ones marked with green.

Reply:
You changed the code to this:

```
 <warning>
  <title>Do Not Use Minibroker on Production Systems</title>
  <para>
   The services deployed by Minibroker are sourced from the stable upstream
   charts repository, see
   <link xlink:href="https://github.com/helm/charts/tree/master/stable"/>,
   and maintained by contributors to the Helm project. Though &suse; supports
   Minibroker itself, it does not support the service charts it deploys.
   Operators should inspect the charts and images exposed by the service plans
   before deciding to use them in a production environment.
  </para>
 </important>
```

and are causing a validation error. Either use <warning></warning> or <important></important>, but not a mixture of both.